readme.md

```diff
@@ -1,4 +1,17 @@
 # Maestro 2.0.0 Alpha
 Maestro 2 is the next version of the Maestro Co-simulation Orchestration Engine.
 
-Please see https://into-cps-maestro.readthedocs.io/ for more information.
\ No newline at end of file
+Please see the [docs](https://into-cps-maestro.readthedocs.io/) for more information.
```

Comment:
Please see [docs](https://into-cps-maestro.readthedocs.io/) , [Chat](https://gitter.im/INTO-CPS/maestro2) for more information.